src/app/login/page.tsx

```diff
@@ -1,9 +1,10 @@
 "use client"
 
-import { useState, useEffect } from 'react';
+import { useState } from 'react';
 import { useMutation, gql } from '@apollo/client';
 import Spinner from '@components/spinner';
 import { useRouter } from 'next/navigation'
+import Notify from '../../config/notiflix-config';
```

Comment:
The only change to do is this. We have import shortcut to make it easier and cleaner to do that kind of things. For the anything in the config folder you can do `@config/...`. So `.../.../config/notiflix-config` would be `@config/notiflix-config`

Reply:
Ohhhh. I did not know about that. Thank you